### PR TITLE
feature/37d changesets for webpack config

### PR DESCRIPTION
**Pull request by reakaleek**
**How to test (feature)**

**How to test (potential regressions)**

### CODE REVIEW

**Comment by reakaleek** [comment]
jenkins test this please

**Comment by reakaleek** [comment]
jenkins test this please

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
   ],
   "scripts": {
     "format": "prettier --loglevel warn --write \"**/*.{js,ts,tsx,css,md}\"",
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "echo \"No test specified\"",
     "build": "rimraf ./dist && tsc",
     "preversion": "npm run build && git commit --no-verify --allow-empty -am \"Update dist\"",
```
Comment (bebraw) [inline package.json]: Let's drop `preversion` as it's redundant now as far as I understand.

**Review by TkDodo — APPROVED** [review]

**Review by bebraw — CHANGES_REQUESTED** [review]
Where is it going to write the change log on release? Will it commit that on merge?

I noticed one thing to change.

**Comment by reakaleek** [comment]
> Where is it going to write the change log on release? Will it commit that on merge?

so.. the basic idea is.. that this pipeline will create a release branch

on that branch it will collect all changesets and post the release notes on github.. and then all changesets are deleted.

After that it will get merged into master and develop

File: package.json (hunk 1 above)
Comment (reakaleek) [reply package.json]: https://github.com/pmedianetwork/webpack-config/pull/9/commits/6508f73afc06e651e83e5a3d2b973c147b199cd6

**Review by bebraw — APPROVED** [review]

**Pull request merged**